test/generate-testcase.js

```diff
@@ -8,10 +8,9 @@ var http = require("http");
 var urlparse = require("url").parse;
 var htmltidy = require("htmltidy2").tidy;
 
-var readabilityCheck = require("../Readability-readerable");
-var readability = require("../index");
-var Readability = readability.Readability;
-var JSDOMParser = readability.JSDOMParser;
+var readabilityCheck = require("../index").isProbablyReaderable;
+var Readability = require("../index").Readability;
```

Comment:
Wouldn't it be preferable to `require` only once? I guess it doesn't matter much as it's just a developer script, but if we're changing it anyway... I *think* these versions of node support destructuring assignment, so maybe we can just do:

```js
var { Readability, isProbablyReaderable } = require("../index");
```
and use `isProbablyReaderable` as a name in the one place where we're using it (which you're having to update anyway, in the next hunk) ?

Reply:
Yes, seems object destructuring [has been available in Node 6+](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment), so can use that. I've tried to touch the code as little as possible in the PR, but you're right this is a good time to switch.

Reply:
This now breaks at linting stage, because eslint defaults to ES5. Since destructuring is only used in tests, I'm inclined to change it back to ES5.